css/src/atomics/colors.scss

```diff
@@ -0,0 +1,213 @@
+// Framework colors
+
+@each $name, $color-set in $colors {
+	$base: nth($color-set, $effect-index-base);
+	$background: nth($color-set, $effect-index-background);
+	$dark: nth($color-set, $effect-index-dark);
+	$hover: nth($color-set, $effect-index-hover);
+	$active: nth($color-set, $effect-index-active);
+	$invert: nth($color-set, $effect-index-invert);
+
+	.text-color-#{$name} {
```

Comment:
Although I can understand not wanting to use `color` only, since `color` is the most poorly named CSS property. I think if we're going to use a pattern, we need to stick with it => `.color-#{$name} {}` etc.

Reply:
Updated